.github/workflows/buildtest.yml

```diff
@@ -1,6 +1,10 @@
 #Originally from https://raw.githubusercontent.com/intel/multus-cni/master/.github/workflows/go-build-test-amd64.yml 
 name: Go-build-and-test-amd64
-on: [push, pull_request]
+on:
+    push:
+    pull_request:
+    schedule:
+      - cron: "0 10 * * *" # everyday at 10 am
```

Comment:
Do we want this cron job in OCP?

Reply:
we don't need this, but I think it's not harmful as we don't have github action enabled in ocp repos.
I just cherry-pick the every commit from upstream, in case there will be conflict in the future.

Reply:
ACK